released/packages/coq-mathcomp-real-closed/coq-mathcomp-real-closed.1.1.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+name: "coq-mathcomp-real-closed"
+maintainer: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
+
+homepage: "https://github.com/math-comp/real-closed"
+bug-reports: "https://github.com/math-comp/real-closed/issues"
+dev-repo: "git+https://github.com/math-comp/real-closed.git"
+license: "CeCILL-B"
+
+build: [ make "-j" "%{jobs}%" ]
+install: [ make "install" ]
+depends: [
+  "coq" { (>= "8.7" & < "8.13~") }
+  "coq-mathcomp-field"       {(>= "1.11.0" & <= "1.12~") | (= "dev")}
```

Comment:
```suggestion
  "coq" { (>= "8.7" & < "8.13~") | (= "dev")}
  "coq-mathcomp-field"       {(>= "1.11.0" & <= "1.12~")}
```
I thought the consensus was to allow `dev` versions of Coq, but not `dev` versions of MathComp dependencies? Otherwise, I would expect there to be PR as soon as this package breaks with `coq-mathcomp-field.dev`. Is something the MathComp team can commit to do?

Reply:
This was a mistake from me, I misplaced the `= "dev"` on the wrong line...
However, I intend to setup the CI so that we can indeed keep this "dev" until it breaks, but I will not commit to that before the CI is indeed testing it.